packages/block-editor/src/store/reducer.js

```diff
@@ -1962,6 +1962,24 @@ export function registeredInserterMediaCategories( state = [], action ) {
 		case 'REGISTER_INSERTER_MEDIA_CATEGORY':
 			return [ ...state, action.category ];
 	}
+
+	return state;
+}
+
+/**
+ * Reducer setting last focused element
+ *
+ * @param {boolean} state  Current state.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {boolean} Updated state.
+ */
+export function lastFocus( state = false, action ) {
+	switch ( action.type ) {
+		case 'LAST_FOCUS':
+			return action.lastFocus;
```

Comment:
Storing unserializable values in Redux's store breaks the compatibility of Redux Devtools. It will just output `t.toString() is not a function` whenever it tries to serialize it and provide no helpful trace 😅.

What's worse is that it could just crash the page unexpectedly, without the Redux Devtools even being active (it just has to be initialized).

Related: https://github.com/WordPress/gutenberg/pull/57198#discussion_r1432826499 and https://github.com/WordPress/gutenberg/pull/57257#issuecomment-1873589846.

Reply:
Thank you for tracking this down! Got an approach for review here: https://github.com/WordPress/gutenberg/pull/57557